docs/open-source-tizen/tools/mic/mic-customize-image.md

```diff
@@ -1,49 +1,55 @@
-# How to Customize Images
+# Customizing Images
```

Comment:
Whenever you change the main tile of the page, you must change TOC.md.

I won't mention it again, whenever you find it, please check TOC.md.

Reply:
We will update the TOC.md file.

However, please clarify:
Is the TOC file supposed to have the same identical titles as the topics themselves?
Or is the TOC file supposed to contain shortened title forms more suitable to limited space in a menu?
Because the current TOC file version seems to have shortened titles...

Reply:
LNB(Left Navigation Bar) area is narrow. If you can find more shorten and suitable titles, you can use them.

Reply:
OK